### DISCUSSION TITLE
flatpak for linux distro

### DISCUSSION THREAD

**Discussion by PMalfG** [freeplane/freeplane#324] [Help]
Hello do you think is it possible to provide an official flatpack with last release ?

**Comment by myfirstnameispaul** [comment]
Indeed, the Flathub is at version 1.8.11.

**Comment by fnatter** [comment]
hi,
I think nobody in the freeplane team provides these (I only provide .debs).
Can you ask the contributors: https://github.com/flathub/org.freeplane.App/graphs/contributors ?
Best Regards,
Felix

**Comment by Raketenphysik** [comment]
Could you please provide the latest version on flathub. I am working in a mixed environment (Windows & openSUSE), so I am struggling with version conflicts. I have no idea regarding the extra effort to provide always the latest versions on flathub. At least for me AppImage would be also an alternative - just not my preferred one.
Best rgrds, Mark

EDIT: Sorry, I didn't get the post above. [https://github.com/flathub/org.freeplane.App/pull/35](https://github.com/flathub/org.freeplane.App/pull/35)
